kafka-client/src/main/java/dev/responsive/kafka/store/ResponsiveStoreRegistry.java

```diff
@@ -54,6 +54,10 @@ public synchronized void deregisterStore(final ResponsiveStoreRegistration regis
     stores.remove(registration);
   }
 
+  public synchronized List<ResponsiveStoreRegistration> stores() {
+    return stores;
```

Comment:
I don't love having to expose this just for the TTD, we might want to take stock and try to clean up the commit logic at some point but I think it's fine for now